README.md

```diff
@@ -1,7 +1,100 @@
-# swift-bson
+# Swift BSON
 
-This is a work-in-progress pure Swift BSON library.
+![swift-bson-logo](etc/swiftBSON.png)
 
+The official MongoDB BSON implementation in Swift!
 
-### Note
-this repository previously contained Swift bindings for libbson, the MongoDB C driver's BSON library. Those bindings are still available under the 2.1.0 tag. Major version 3.0 and up will be used for the BSON library.
+## Index
+
+- [Documentation](#documentation)
+- [Bugs/Feature Requests](#bugs--feature-requests)
+- [Installation](#installation)
+- [Example Usage](#example-usage)
+  - [Work With and Modify Documents](#work-with-and-modify-documents)
+- [Development Instructions](#development-instructions)
+
+## Documentation
+
+The latest documentation is available [here](https://mongodb.github.io/swift-bson/).
+
+## Bugs / Feature Requests
+
+Think you've found a bug? Want to see a new feature in `swift-bson`? Please open a case in our issue management tool, JIRA:
+
+1. Create an account and login: [jira.mongodb.org](https://jira.mongodb.org)
+2. Navigate to the SWIFT project: [jira.mongodb.org/browse/SWIFT](https://jira.mongodb.org/browse/SWIFT)
+3. Click **Create Issue** - Please provide as much information as possible about the issue and how to reproduce it.
+
+Bug reports in JIRA for all driver projects (i.e. NODE, PYTHON, CSHARP, JAVA) and the Core Server (i.e. SERVER) project are **public**.
+
+## Installation
+
+The library supports use with Swift 5.1+. The minimum macOS version required to build the library is 10.14. The library is tested in continuous integration against macOS 10.14, Ubuntu 16.04, and Ubuntu 18.04.
+
+Installation is supported via [Swift Package Manager](https://swift.org/package-manager/).
+
+### Install BSON
+
+The driver contains two modules to support a variety of use cases: an asynchronous API in `MongoSwift`, and a synchronous API in `MongoSwiftSync`. The modules share a BSON implementation and a number of core types such as options `struct`s.
+
+To install the driver, add the package and relevant module as a dependency in your project's `Package.swift` file:
```

Comment:
this sentence should also be updated, maybe just "driver" -> "library" and drop the "relevant" since there is only one module